packages/manager/modules/pci/src/projects/project/instances/instances.html

```diff
@@ -189,7 +189,7 @@
                 ></span>
             </oui-action-menu-item>
             <oui-action-menu-item
-                data-ng-if="!$row.hasPublicIpV4() && !$row.hasPublicIpV6() && $ctrl.isAdditionalIpsAvailable"
+                data-disabled="$row.hasPublicIpV4() && $row.hasPublicIpV6() && $ctrl.isAdditionalIpsAvailable"
```

Comment:
**nitpick:** Should be displayed only if Floating IP is supported in the region.
```suggestion
                data-ng-if=" $ctrl.isAdditionalIpsAvailable"
                data-disabled="$row.hasPublicIpV4() && $row.hasPublicIpV6()"
```